### ISSUE TITLE
Error with installation

### ISSUE THREAD

**Issue by GinoWoz1** [lacava/few#39]
Hello,

Trying to attempt this package but running into some issues, any idea? I have VS 14.16 now on PC and getting this error when typing 'pip install few'. At first it was asking for eigency but now after that installation this error popped up.

![image](https://user-images.githubusercontent.com/17535345/48670205-86c1b280-eac8-11e8-905f-2000238fa421.png)

Sincerely,
G

**Comment by lacava** [comment]
Congrats, I think you're the first person I've talked to test the windows install! 

I think the issue is that the flag to specify the c++ standard to compile with is different in VS than the GNU compiler. Could you try this and let me know if it fixes it: 

clone the repo directly rather than installing via pip. from the code directory, change  line 79 of setup.py to 

```
extra_compile_args = ['/std:c++latest'])],
```

then from the code directory run

```
python setup.py install
```

to install it. This should work.

**Comment by GinoWoz1** [comment]
Thanks getting more errors now. The unrecognized issue is not showing up but now getting eigency errors.

![image](https://user-images.githubusercontent.com/17535345/48733693-41040780-ebf8-11e8-9f9b-00a14730330d.png)
![image](https://user-images.githubusercontent.com/17535345/48733815-89bbc080-ebf8-11e8-80f4-b2f4f8a6d46c.png)

![image](https://user-images.githubusercontent.com/17535345/48733844-9d672700-ebf8-11e8-91f8-55950e1122e5.png)

**Comment by lacava** [comment]
I've pushed an updated version of Few to pypi (FEW-0.0.51) that should address the OS issue with `extra_compile_args` and the iota error and random_shuffle error. 

I'm not sure what is going on with the eigency errors. There is a newer stable release of Eigen (3.3.5) and there is an option in eigency to point to a local copy of Eigen. that might work if my changes don't make those errors magically go away.

**Comment by GinoWoz1** [comment]
Thanks! So is that available now?

**Comment by lacava** [comment]
yep, you can clone or install via pip

**Comment by GinoWoz1** [comment]
Thanks alot @lacava . You can close this out. 

Quick question, how do I ask FEW to minimize? it is seemingly trying to maximize the loss score. I am using RMSLE so ideally I should be minimizing this.

Sincerely,
G

**Comment by lacava** [comment]
there are two different loss fns used by FEW: the one that assigns fitness to the features in the model (controlled by `self.fit_choice`) and the one that keeps track of the best model so far, controlled by `self.scoring_function`. 

`self.scoring_function` has to be maximized. But you can pass in a scoring function that inverts the RMSLE. I haven't tried this but something like this should work

```python
def minusRMSLE(y, y_pred): 
    return - RMSLE( y, y_pred ) 

est = FEW(scoring_function = minusRMSLE)
``` 

you can also pass [any of these](https://scikit-learn.org/stable/modules/classes.html#sklearn-metrics-metrics)

**Comment by GinoWoz1** [comment]
Thanks. I tried it but my initial CV score is almost my best that I got, by 100 generations the score has gotten significantly worse. I will open another issue on questions regarding current ml validation score.
